### ISSUE TITLE
QueueableJob Naming Convention

### ISSUE THREAD

**Issue by yasalkhatib** [lucid-architecture/laravel#66]
### Problem:
Currently, when creating a queueable job using the command: 
`make:job --queue LogUserSearch user`
The job class name will be as follows: **LogUserSearchJob**

### Suggestion:
I would suggest changing the naming convention of queueable jobs to something like <job>QueueableJob or <job>QJob (LogUserSearchQueueableJob, or LogUserSearchQJob).
**By this,** the code will be more readable and clear. In addition, the developers will be aware that this job is queuable and hence it will run asynchronously.

**Comment by Mulkave** [comment]
Good idea! Will trt to collect some votes on this from our [Slack workspace](https://lucid-slack.herokuapp.com) and move this to [Ideas issue board](https://github.com/lucid-architecture/ideas).
